src/test/java/net/pms/formats/FormatsRegressionTest0.java

```diff
@@ -47,7 +47,7 @@ public void test05() throws Throwable {
 	public void test07() throws Throwable {
 		MKV mKV0 = new MKV();
 		String str1 = mKV0.toString();
-		String[] str_array2 = mKV0.getId();
+		String[] str_array2 = mKV0.getSupportedExtensions();
```

Comment:
I'm confused about this change, but haven't looked into it so it might make sense. I am just leaving notes :)

Reply:
@SubJunk removed deprecated method `getId()` from `Format`  class which returned `getSupportedExtensions()`